base/MAPL_Generic.F90

```diff
@@ -1379,7 +1380,8 @@ recursive subroutine MAPL_GenericInitialize ( GC, IMPORT, EXPORT, CLOCK, RC )
       _VERIFY(STATUS)
 
       DO  I = 1, NRA
-         AlarmName = "RecordAlarm" // CHAR(I+ICHAR('0')) 
+         write(alarmNum,'(I3.3)') I
```

Comment:
Is 3 digits enough?    We've generally been trying to aim for an order of magnitude beyond any use cases that we have now.  3 is probably enough, but I figured I'd make you think about it to be certain.

Reply:
3 digit goes way above all use cases currently cover (the "old" limit was a single digit, in GEOS we never exceeded it, only Lizzie currently ran into this trying to use this for 12 month worth of alarm). We have the ability to use frequency to specify these alarms (this is the typical usage) and these are still 1 record alarm. In addition, we have the ability to specify specific date/times at arbitrary points, and this is what Lizzie exceeded.